### DISCUSSION TITLE
Renaming projects on the testbeats dashboard

### DISCUSSION THREAD

**Discussion by kamauvick** [test-results-reporter/reporter#231] [Q&A] [answered]
Hello, 
Is there a provision to edit the project name on testbeats, I have a project with quite a number of runs to it and I needed to update the name.

**Comment by ASaiAnudeep** [comment, marked as answer]
Hello @kamauvick , the capability to update project details is under development. We will update this thread once it is available. You can expect this to be available by next week.

**Reply by ASaiAnudeep** [reply]
Hello @kamauvick , the changes are deployed. We should be able to update the project name at https://app.testbeats.com/settings/projects

> Make sure to update your config file.
